setup.py

```diff
@@ -35,15 +35,13 @@
         "datasets>=1.2.1",
         "evaluate",
         "protobuf==3.20.1",
-        "diffusers",
     ],
     "onnxruntime-gpu": [
         "onnx",
         "onnxruntime-gpu>=1.9.0",
         "datasets>=1.2.1",
         "evaluate",
         "protobuf==3.20.1",
-        "diffusers",
```

Comment:
Shouldn't we remove this dependency from `"exporters-tf"` and `"exporters"` as well ?

Reply:
yes